recipes/curio/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "curio" %}
+{% set version = "0.4" %}
+{% set sha256 = "8e241a9d1958d85f53a2549fe948149abb43dbaac140f1cf2a9979856dc7c364" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://github.com/dabeaz/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  skip: True  # [win or not py35]
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
```

Comment:
Hi @synapticarbors, I think that is pretty much redundant since the first thing the test suite does is import curio. Also, many packages cannot use the import key because they are not Python. I am happy to add you as a maintainer if this is something you would like to do in the feedstock.